CAP/gap/MethodRecord.gi

```diff
@@ -2240,7 +2240,7 @@ AstrictionToCoimageWithGivenCoimage := rec(
 
 UniversalMorphismIntoCoimage := rec(
   installation_name := "UniversalMorphismIntoCoimage",
-  filter_list := [ "morphism", IsList ],
+  filter_list := [ "morphism", "list_of_morphisms" ],
```

Comment:
Depending on your application of this new specification, you may have to be careful here. This list of morphisms is not of arbitrary length, but has to be of length 2 (it stands for a possible factorization of the first argument).

Reply:
You are right, I have reverted the four occurrences. Thanks for noticing!